src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -133,7 +134,8 @@
         (when editing?
           [comps/SidebarButton {:color :exception-state :margin :top
                                 :text "Cancel Editing" :icon :cancel
-                                :onClick #(stop-editing state)}])]))])
+                                :onClick #(do (react/call :load-new-method-template this (:prev-snapshot-id @state))
```

Comment:
3. It's entirely possible I'm misreading this (not running it right now), but couldn't you just use the `snapshot-id` you `let` previously and skip this `:prev-snapshot-id` thing entirely?